### PR TITLE
Add support for XML heredoc and nowdoc

### PR DESCRIPTION
**Pull request by akrabat**
Use XML syntax highlighting within `<<<XML` and `<<<'XML'` heredoc/nowdoc strings.

### CODE REVIEW

File: syntax/php.vim [hunk 1]
```diff
@@ -158,6 +160,21 @@ if ((exists("php_sql_query") && php_sql_query) || (exists("php_sql_heredoc") &&
   endif
 endif
 
+if !exists("php_xml_heredoc")
+  let php_xml_heredoc=1
```
Comment (StanAngeloff) [inline syntax/php.vim]:
```suggestion
  let php_xml_heredoc=0
```

Can we make this off by default?

File: syntax/php.vim [hunk 2]
```diff
@@ -158,6 +160,21 @@ if ((exists("php_sql_query") && php_sql_query) || (exists("php_sql_heredoc") &&
...
+endif
+
+if !exists("php_xml_nowdoc")
+  let php_xml_nowdoc=1
```
Comment (StanAngeloff) [inline syntax/php.vim]:
```suggestion
  let php_xml_nowdoc=0
```

Same, can we make this off by default?

File: syntax/php.vim (hunk 1 above)
Comment (StanAngeloff) [inline syntax/php.vim]:
```suggestion
  let php_xml_heredoc=0
```

Can we make this off by default?

File: syntax/php.vim [hunk 3]
```diff
@@ -37,6 +37,8 @@
 "           php_sql_nowdoc = 1 for SQL syntax highlighting inside nowdocs (default: 1)
 "           b:sql_type_override = 'postgresql' for PostgreSQL syntax highlighting in current buffer (default: 'mysql')
 "           g:sql_type_default = 'postgresql' to set global SQL syntax highlighting language default (default: 'mysql')"
+"           php_xml_heredoc = 1 for XML syntax highlighting inside heredocs (default: 1)
```
Comment (StanAngeloff) [inline syntax/php.vim]:
```suggestion
"           php_xml_heredoc = 1 for XML syntax highlighting inside heredocs (default: 0)
```

File: syntax/php.vim [hunk 4]
```diff
@@ -37,6 +37,8 @@
...
+"           php_xml_nowdoc = 1 for XML syntax highlighting inside nowdocs (default: 1)
```
Comment (StanAngeloff) [inline syntax/php.vim]:
```suggestion
"           php_xml_nowdoc = 1 for XML syntax highlighting inside nowdocs (default: 0)
```

File: syntax/php.vim (hunk 2 above)
Comment (akrabat) [reply syntax/php.vim]: Certainly.

**Comment by StanAngeloff** [comment]
Thank you!

**Pull request merged**